test/system/my/track_test.rb

```diff
@@ -68,6 +82,21 @@ class My::TrackTest < ApplicationSystemTestCase
     })
 
     visit track_path(track)
-    within(".main-section") { assert_text "Mentored Mode (Recommended)" }
+    within(".main-section") { assert_text "Mentored Mode" }
+  end
+
+  test "shows oversubscribed migration modal for user tracks created before migration" do
```

Comment:
Do tracks before migration behave differently so we need this test?

Reply:
Yes. There is a different modal for migrating to tracks and for signing up. The test file already had both in, so I added an extra one for each.